src/App.tsx

```diff
@@ -1,64 +1,63 @@
+import ResetPass from "./Modules/Authontication/Components/ResetPass/ResetPass";
+import Register from "./Modules/Authontication/Components/Register/Register";
+import VerifyAccount from "./Modules/Authontication/Components/VerifyAccount/VerifyAccount";
+import ChangePass from "./Modules/Authontication/Components/ChangePass/ChangePass";
+import MasterLayout from "./Modules/Shared/Components/MasterLayout/MasterLayout";
+import Dashboard from "./Modules/Dashboard/Dashboard";
+import ProjectList from "./Modules/Projects/Components/ProjectList/ProjectList";
+import ProjectData from "./Modules/Projects/Components/ProjectData/ProjectData";
+import TasksList from "./Modules/Tasks/Components/TasksList/TasksList";
+import TasksData from "./Modules/Tasks/Components/TasksData/TasksData";
+import UsersList from "./Modules/Users/Components/UsersList/UsersList";
+import NavBar from "./Modules/Shared/Components/NavBar/NavBar";
+import SideBar from "./Modules/Shared/Components/SideBar/SideBar";
+import { ToastContainer } from "react-toastify";
 
 function App() {
-  
   const routes = createBrowserRouter([
     {
-        path:'',
-        element:<AuthLayout/>,
-        errorElement:<NotFound/>,
-        children:[
-          {index:true, element: <LogIn/>},
-          {path:'login', element: <LogIn/>},
-          {path:'forget-pass', element: <ForgetPass/>},
-          {path:'reset-pass', element: <ResetPass/>},
-          {path: 'register' , element:<Register/>},
-          {path: 'change-pass' , element:<ChangePass/>},
-          {path: 'verify-account' , element:<VerifyAccount/>}
-        ]
+      path: "",
+      element: <AuthLayout />,
+      errorElement: <NotFound />,
+      children: [
+        { index: true, element: <LogIn /> },
+        { path: "login", element: <LogIn /> },
+        { path: "forget-pass", element: <ForgetPass /> },
+        { path: "reset-pass", element: <ResetPass /> },
+        { path: "register", element: <Register /> },
+        { path: "change-pass", element: <ChangePass /> },
+        { path: "verify-account", element: <VerifyAccount /> },
+      ],
     },
     {
-      path:'dashboard',
-      element: <MasterLayout/>,
-      errorElement:<NotFound/>,
-      children:[
-        {index:true, element: <Dashboard/>},
-        {path:'home', element: <Dashboard/>},
-        {path:'project-list', element: <ProjectList/>},
-        {path:'project-data', element: <ProjectData/>},
-        {path:'tasks-list', element: <TasksList/>},
-        {path:'tasks-data', element: <TasksData/>},
-        {path:'users-list', element: <UsersList/>},
-        
-       
-      ]
-    }
-  ])
-
+      path: "dashboard",
+      element: <MasterLayout />,
+      errorElement: <NotFound />,
+      children: [
+        { index: true, element: <Dashboard /> },
+        { path: "home", element: <Dashboard /> },
+        { path: "project-list", element: <ProjectList /> },
+        { path: "project-data", element: <ProjectData /> },
```

Comment:
If this component is used for the new project form, I suggest renaming the path to `projects/new-project`. This will indicate that the new-project page is reached from the projects page and it will be more meaningful.

Reply:
This component is actually used for CRUD operations on projects